client/src/components/Property.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable immutable/no-mutation */
+import React, {Component,Fragment} from 'react';
+import { withApollo } from 'react-apollo';
+import AutoComplete from './AutocompleteText';
+import Pagination from './Pagination';
+import Properties from './Properties';
+import { prevquery, nextquery } from '../queries/searchquery';
+export class Property extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+          filter: '',
+          properties : [],
+          items : '',
+        }
+        this.paginate = {
+           page : 1,
+           last : 5,
+           first : 5,
+          cursor : {
+            before : '',
+            after : ''
+          },
+           hasNextPage : true,
+           hasPreviousPage : false
+        }
+      this.selected.bind(this);
+      this.pagination = this.pagination.bind(this);
+      this._search("");
```

Comment:
Data fetching e.g. `this._search("")`, should be executed after the component was loaded. Put the logic in the correct component lifecycle.

You can read it here: https://reactjs.org/docs/react-component.html#the-component-lifecycle